doc/installation/system/inifile.rst

```diff
@@ -226,13 +226,17 @@ privacyIDEA Nodes
 -----------------
 
 privacyIDEA can run in a redundant setup. For statistics and monitoring purposes you
-can give these different nodes, dedicated names.
+can give these different nodes dedicated names.
 
-``PI_NODE`` is a string with the name of this very node. ``PI_NODES`` is a list of
-all available nodes in the cluster.
+``PI_NODE`` is a string with the name of this very node. At the startup of
+the application, an installation specific unique ID will be used to tie the
```

Comment:
For me it would be more clear, that it is the startup of "privacyIDEA"; not the application.

Reply:
Done in c0c5f2d5e